### PR TITLE
Added  'is-rounded' modifier supprt for <Image />, closes #113

### LINKED ISSUE #113: Rounded Images
Hey,

As per https://bulma.io/documentation/elements/image/#rounded-images I would like to request a such a property on the `<Image />` react component.

Currently using `is-rounded` on the `Image` will not yield desired result, as the class needs to be applied to the wrapped ``.

Cheers 🍺

### PR DESCRIPTION
**Pull request by MrCreeper1008**
Doc for `is-rounded` modifier: https://bulma.io/documentation/elements/image/#rounded-images  

New `boolean` prop: `rounded` in component `<Image />`.  

Related issue: #113

### CODE REVIEW

File: src/components/image/image.js [hunk 1]
```diff
@@ -62,7 +65,14 @@ export default class Image extends PureComponent {
           [`is-${s}`]: s,
         })}
       >
-        <img onError={this.onError} src={this.state.src} alt={alt} />
+        <img
+          className={classnames(className, {
```
Comment (couds) [inline src/components/image/image.js]: I'm not sure about adding the same classmates to the figure and the img tag. Maybe just adding the  is-rounded will be enough. Wdyt?

Comment (MrCreeper1008) [reply src/components/image/image.js]: @couds yes you're right; will fix asap.

**Comment by MrCreeper1008** [comment]
What is up with the weird `404 not found flatmap-stream@0.1.1` error??

**Comment by MrCreeper1008** [comment]
Related issue https://github.com/dominictarr/event-stream/issues/115

**Comment by MrCreeper1008** [comment]
Which leads to a more serious issue: https://github.com/dominictarr/event-stream/issues/116

**Pull request merged**